### PR TITLE
[lgwebos] Fix items example

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: bundles/org.openhab.binding.lgwebos/README.md [hunk 1]
```diff
@@ -93,16 +93,20 @@ A sample HABPanel remote control widget can be found [in this github repository.
 
 ## Example
 
-Assuming your ThingID is lgwebos:WebOSTV:3aab9eea-953b-4272-bdbd-f0cd0ecf4a46.
+demo.items:
```
Comment (J-N-K) [inline bundles/org.openhab.binding.lgwebos/README.md]:
```suggestion
demo.things:
```

Comment (lolodomo) [reply bundles/org.openhab.binding.lgwebos/README.md]: Fixed

[the remark this answers is not in the document]
Comment (lolodomo) [reply bundles/org.openhab.binding.lgwebos/README.md]: This is already fixed. This was so stupid...

**Comment by J-N-K** [comment]
Manually verified sign.off

**Pull request merged**